src/healthtable_interface.jl

```diff
@@ -0,0 +1,16 @@
+using Tables
+using Base: @kwdef
+
+@kwdef struct HealthTable{T}
+    source::T
+    omop_cdm_version::String
+end
+
+Tables.istable(::Type{<:HealthTable}) = true
+Tables.rowaccess(::Type{<:HealthTable}) = true
+Tables.rows(ht::HealthTable) = Tables.rows(ht.source)
+Tables.columns(ht::HealthTable) = Tables.columns(ht.source)
+Tables.schema(ht::HealthTable) = Tables.schema(ht.source)
+Tables.materializer(::Type{<:HealthTable}) = DataFrame
```

Comment:
Could you add some docstrings to these functions? Maybe take a look at Tables.jl to see how they may document these functions... I am not so sure myself how they would do it.

Also, do you need to import these functions from Tables.jl to do this kind of type piracy?